packages/cli/src/util/createServer.ts

```diff
@@ -22,23 +22,32 @@ export async function createMultiProcessPrism(options: CreatePrismOptions) {
     }
   } else {
     const logInstance = createLogger('CLI');
-    return createPrismServerWithLogger(options, logInstance);
+    try {
+      return await createPrismServerWithLogger(options, logInstance);
```

Comment:
In a return statement, await is redundant.

```suggestion
      return createPrismServerWithLogger(options, logInstance);
```